### PR TITLE
updated changelog

### PR DESCRIPTION
**Pull request by David-OConnor**
Will add a [migration guide](https://github.com/seed-rs/seed/issues/345) to the readme before merge

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,9 @@
 # Changelog
 
 [unreleased]
+- (placeholder)
 
+## v0.6.1
```
Comment (MartinKavik) [inline CHANGELOG.md]: 0.6.1 vs 0.6.0

**Comment by David-OConnor** [comment]
Ready when you are

**Comment by MartinKavik** [comment]
Do you want to add changes required by release checklist to this PR?

**Comment by MartinKavik** [comment]
If so, remaining steps are:
- `2. Update the version of seed dependency in the readme.`
- `8. Run cargo make populate_all to synchronize St, At and other enums with official values` (maybe you already run it)

And don't forget to update to new `wasm-pack`, Rust and `cargo-make`. Before running `cargo make verify` and other commands.

**Comment by David-OConnor** [comment]
Here are the breaking ones:

- [BREAKING] Removed deprecated `update` and `trigger_update_ev`.
Already deprecated

- [BREAKING] Removed the remains of lifecycle hooks.
An app that uses these hooks needs to restructure this part; there's no quick fix

- [BREAKING] Removed unpredictable internal input listeners - Seed will not longer react to external input value changes.
I suspect that code this breaks will be an indication of a messy procedure that needs to be restructured.

- [BREAKING] Use `EventHandler` instead of `Listener`. (`Listener` is now used as the internal DOM EventListener representation.)
Simple swap

- [BREAKING] Added or changed `Custom` variant from `Custom(String)` to `Custom(Cow<'static, str>)` 
I suspect compiler errors will be constructive here.

**Comment by MartinKavik** [comment]
Yeah, nice migration guide :)
And new features like `ElRef`, `nodes!`, `El::custom`, etc. should be documented in seed-rs.org.

**Comment by David-OConnor** [comment]
https://github.com/seed-rs/seed-rs.org/pull/27

**Comment by MartinKavik** [comment]
@David-OConnor what's the release status, pls?

**Comment by David-OConnor** [comment]
Ready to roll

**Pull request merged**